tests/datasets/configuration/test_storage_loader.py

```diff
@@ -134,6 +136,8 @@ class TestStorageConfiguration(ConfigurationTest):
         metrics_distributions_storage,
         metrics_raw,
         org_counters_storage,
+        outcomes_hourly,
+        outcomes_raw,
```

Comment:
while we're here can we fix this test? If the storage never existed in code (which new ones won't) it won't work

https://github.com/getsentry/snuba/pull/3468#discussion_r1041551487

Reply:
I've removed that check entirely since it assumes the only config storages are ones that have a python version.